### PR TITLE
DrivAerML: Fourier+no-EMA — longer cosine schedule T_max=100/300

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis
The current best DrivAerML result (PR #2475, val=51.35%) uses T_max=30 with only 2 epochs completed at cutoff. With a longer training budget (180-min timeout), T_max=30 may cause the LR to hit near-zero too early, starving later epochs. A longer cosine schedule (T_max=100 or T_max=300) may better match the actual number of epochs achieved, keeping the LR in a productive range throughout training.

## Instructions

Run two parallel jobs on both GPUs:

**Run 1 — T_max=100:**
```bash
cd target/icml2026 && CUDA_VISIBLE_DEVICES=0 python train.py \
  --dataset drivaerml \
  --optimizer adamw \
  --lr 5e-4 \
  --cosine_t_max 100 \
  --no-use-ema \
  --enable-fourier \
  --epochs 999 \
  --batch-size 1 \
  --drivaerml-train-surface-points 50000 \
  --drivaerml-eval-surface-points 50000 \
  --max-train-batches 394 \
  --max-eval-batches 200 \
  --wandb_group chrome-drivaerml-tmax-sweep &
```

**Run 2 — T_max=300:**
```bash
cd target/icml2026 && CUDA_VISIBLE_DEVICES=1 python train.py \
  --dataset drivaerml \
  --optimizer adamw \
  --lr 5e-4 \
  --cosine_t_max 300 \
  --no-use-ema \
  --enable-fourier \
  --epochs 999 \
  --batch-size 1 \
  --drivaerml-train-surface-points 50000 \
  --drivaerml-eval-surface-points 50000 \
  --max-train-batches 394 \
  --max-eval-batches 200 \
  --wandb_group chrome-drivaerml-tmax-sweep
```

Wait for both to complete. Report `val_primary/surface_rel_l2_pct` for each run, the number of epochs completed, and the final LR at cutoff. Note which T_max produces a better result and whether the LR was still in a productive range at the end.

**Critical:** You must pass `--epochs 999` — the default is only 2 epochs due to a hardcoded bug in the training script.

## Baseline
- **Primary metric:** `val_primary/surface_rel_l2_pct` (lower is better)
- **Current best (val):** 51.35% (PR #2475, W&B run: 5ncrjm32)
- **Current best (test):** 52.06%
- **Reproduce baseline:** `cd target/icml2026 && python train.py --dataset drivaerml --optimizer adamw --lr 5e-4 --cosine_t_max 30 --no-use-ema --enable-fourier --epochs 999 --batch-size 1 --drivaerml-train-surface-points 50000 --drivaerml-eval-surface-points 50000 --max-train-batches 394 --max-eval-batches 200`

### CODE REVIEW

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2525 because Duplicate assignment — closing extra PR created by parallel agent. Student already has a primary experiment assigned..

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2525 because Closing duplicate — student chrome already has active PR #2515 assigned..
